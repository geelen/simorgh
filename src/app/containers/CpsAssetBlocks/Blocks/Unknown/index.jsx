import React from 'react';
import { string } from 'prop-types';

/* eslint-disable react/no-danger */
const UnknownBlockContainer = () => (
  <div
    dangerouslySetInnerHTML={{ __html: `<!-- unknown block -->` }}
  />
);

export default UnknownBlockContainer;
