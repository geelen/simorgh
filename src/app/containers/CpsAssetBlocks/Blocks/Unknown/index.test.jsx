import React from 'react';
import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
import Unknown from '.';

describe('CpsAssetBlocks Unknown', () => {
  shouldMatchSnapshot('should render correctly', <Unknown />);
});
