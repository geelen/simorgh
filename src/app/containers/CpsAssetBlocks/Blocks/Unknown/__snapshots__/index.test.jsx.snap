// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaPageBlocks Unknown should render correctly 1`] = `
<div>
  <!-- unknown block -->
</div>
`;
